extensions/wikia/AdEngine/webpack.config.js

```diff
@@ -0,0 +1,60 @@
+const path = require('path');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+const destinationPath = path.resolve(__dirname, 'js/build');
+const cssExtract = new ExtractTextPlugin('styles.scss');
+
+module.exports = [{
+	entry: [
+		path.resolve(__dirname, 'src', 'ad-engine.bridge.js'),
+	],
+	output: {
+		path: destinationPath,
+		filename: 'ad-engine.bridge.js',
+		libraryTarget: 'amd',
+		library: 'ad-engine.bridge'
+	},
+	resolve: {
+		extensions: ['.js']
+	},
+	module: {
+		rules: [
+			{
+				test: /\.jsx?$/,
+				use: {
+					loader: 'babel-loader',
+					options: {
+						presets: ['es2015']
+					}
+				}
+			}
+		]
+	}
+}, {
+	entry: [
+		path.resolve(__dirname, 'src', 'ad-engine.bridge.scss')
+	],
+	output: {
+		// TODO: remove me I'm hack for webpack scss build
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[project-id]: wikia:app

Reply:
Any plans to get rid of this hack?

Reply:
I've tried with couple approaches, but I have no idea how to fix it correctly :(